VimeoNetworking/Sources/ResponseCache.swift

```diff
@@ -163,116 +85,6 @@ final public class ResponseCache
-                }
-                catch let error
-                {
-                    print("ResponseDiskCache error: \(error)")
-                }
-            })
-        }
-        
-        func responseDictionary(forKey key: String, completion: @escaping (VimeoClient.ResponseDictionary?) -> Void)
-        {
-            self.queue.async {
-                
-                guard let filePath = self.fileURL(forKey: key)?.path
-                    else
-                {
-                    assertionFailure("No cache path found.")
-                    
-                    return
-                }
-                
-                guard let data = try? Data(contentsOf: URL(fileURLWithPath: filePath))
-                else
-                {
-                    completion(nil)
-                    
-                    return
-                }
-                
-                var responseDictionary: VimeoClient.ResponseDictionary? = nil
-                
-                do
-                {
-                    try ExceptionCatcher.doUnsafe
-                    {
-                        responseDictionary = NSKeyedUnarchiver.unarchiveObject(with: data) as? VimeoClient.ResponseDictionary
-                    }
-                }
-                catch let error
-                {
-                    print("Error decoding response dictionary: \(error)")
-                }
-                
-                completion(responseDictionary)
-            }
-        }
-        
-        func removeResponseDictionary(forKey key: String)
-        {
-            self.queue.async(flags: .barrier, execute: {
-                
-                let fileManager = FileManager()
-                
-                guard let filePath = self.fileURL(forKey: key)?.path else
-                {
-                    // Assert to catch a badly formed filepath.
-                    
-                    assertionFailure("No valid cache file path found for key: \(key).")
-                    
-                    return
-                }
-                
-                guard fileManager.fileExists(atPath: filePath) == true else
-                {
-                    // Multiple attempts to remove the cache file are possible given that
-                    // a cached response won't be repopulated until another request is made.
-                    // Thus we don't assert, and simply exit the method.
-                    
-                    print("No cache file to remove for key: \(key).")
-                    
-                    return
-                }
-                
-                do
-                {
-                    try fileManager.removeItem(atPath: filePath)
-                }
-                catch let error
-                {
-                    print("Removal of disk cache for \(key) failed with error \(error)")
-                }
-            })
-        }
-        
         func removeAllResponseDictionaries()
```

Comment:
Was wondering if we'd like to create a ticket that will eventually remove the entire class?

Reply:
That's a good idea, yeah. I'll throw it in the epic.